lib/ransack/mongoid/version.rb

```diff
@@ -1,5 +1,5 @@
 module Ransack
   module Mongoid
-    VERSION = "0.1.0"
```

Comment:
I don't think you need to bump the version, since nothing is really changing from a final user perspective. This PR is just making sure that this gem works against newest versions of dependencies.

Reply:
Without the version bump, the gem does not update and it does not resolve the new decency versions.